### ISSUE TITLE
smh

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2409]
<img width="569" alt="CleanShot 2022-01-09 at 13 00 03@2x" src="https://user-images.githubusercontent.com/5903821/148677621-d0eeaca6-8b78-4fbb-b16e-3d7ac24c4c98.png">

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.28.11](https://github.com/open-wa/wa-automate-nodejs/compare/4.28.10...4.28.11) (2022-01-09)

- 🐛  fix #2409 [`#2409`](https://github.com/open-wa/wa-automate-nodejs/issues/2409)
- Release 4.28.11 [`89503ba`](https://github.com/open-wa/wa-automate-nodejs/commit/89503ba3415a115d753dfbd53f29ccc82f668314)
- updated types-only package [`a9b0a3a`](https://github.com/open-wa/wa-automate-nodejs/commit/a9b0a3a20bae2c9eae01ca6498a6e0d51d98e1df)
